apierrors/errors.go

```diff
@@ -6,14 +6,16 @@ import (
 
 // A list of error messages for Topic API
 var (
-	ErrTopicNotFound                  = errors.New("topic not found")
 	ErrContentNotFound                = errors.New("content not found")
-	ErrNotFound                       = errors.New("not found")
+	ErrContentUnrecognisedParameter   = errors.New("content query not recognised")
+	ErrEmptyRequestBody               = errors.New("request body empty")
 	ErrInternalServer                 = errors.New("internal error")
-	ErrUnableToReadMessage            = errors.New("failed to read message body")
-	ErrUnableToParseJSON              = errors.New("failed to parse json body")
+	ErrInvalidReleaseDate             = errors.New("invalid topic release date, must use RFC3339 format")
```

Comment:
(question): Is this error returned in api response ? if so, not sure the consumers would find it meaningful to get a RFC3339 format in the error response, rather we can just say invalid topic release date format ? May be I am wrong but just a thought.

Reply:
I will give an example of the format of the release date instead of stating RFC3339 format